### ISSUE TITLE
addresses are now truncated in answers

### ISSUE THREAD

**Issue by wjwwood** [ros-infrastructure/answers.ros.org#66]
Addresses are being truncated and suffixed with `...`, see:

http://answers.ros.org/question/126781/bloom-release-failed-to-open-pull-request/?answer=126866#post-id-126866

![screen shot 2014-02-06 at 10 34 09 am](https://f.cloud.github.com/assets/100427/2102340/87d8819c-8f5d-11e3-8cd4-87e08e67bf69.png)

The two links on that page are to two different places, but because of the formatting, they appear the same, this is really quite awful, can we disable this?

**Comment by evgenyfadeev** [comment]
The link text is truncated when urls are long, but the links themselves still work.
Maybe they should be truncated somehow in the middle instead?

**Comment by wjwwood** [comment]
If you must truncate them, then yeah, I would do it in the middle, but why do they need to be truncated?

**Comment by evgenyfadeev** [comment]
we truncate very long urls, urls can be up to ~2000 characters long.

**Comment by wjwwood** [comment]
So? Who else truncates urls? Github, Gmail, and Bitbucket don't, they just line wrap them if they are long. I appreciate the intent of the feature was to improve the look and feel, but I don't see the advantage to truncating the urls, if I want them to look nice I can give the link with alternative text rather than just the raw url.

**Comment by evgenyfadeev** [comment]
Let's think about this a little bit, I understand your concern. To address
this somewhat we can increase the maximum length.

On Thu, Feb 6, 2014 at 4:20 PM, William Woodall notifications@github.comwrote:

> So? Who else truncates urls? Github, Gmail, and Bitbucket don't, they just
> line wrap them if they are long. I appreciate the intent of the feature was
> to improve the look and feel, but I don't see the advantage to truncating
> the urls, if I want them to look nice I can give the link with alternative
> text rather than just the raw url.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/ros-infrastructure/answers.ros.org/issues/66#issuecomment-34359735
> .

## 

Askbot
Valparaiso, Chile
skype: evgeny-fadeev

**Comment by wjwwood** [comment]
Thanks, it might be worth adding a syntax option to disable truncating regardless of length, maybe a leading `!` or something like that if you intend to keep the truncating behavior in the long term.

**Comment by NikolausDemmel** [comment]
I agree with @wjwwood here that, while I can understand the intent, for me this is counter productive. I can manually provide a link text if I really want to and prefer to immediately see where a link is going otherwise.

I would vote to disable this feature on answers.ros.org.

**Comment by tfoote** [comment]
Lets try the truncation via the middle of the link instead of the end.

**Comment by wjwwood** [comment]
Bump on this thread.

This still annoys me every time I use answers.ros.org. It just makes it hard to know what you're clicking on at a glance.
